examples/annotation_mapping.ipynb

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
<p>ignore the error messages </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>